include/bio/record.hpp

```diff
@@ -324,7 +331,6 @@ auto const && get(record<field_ids, field_types...> const && r)
 //-------------------------------------------------------------------------------
 
 /*!\name Create a record from the arguments.
- * \relates bio::record
```

Comment:
This doesn't work?

Reply:
This triggers a weird warning that there are multiple `\relates`. But since nothing is in modules right now because `al//hpp` headers with `defgroups` are missing I couldn't quickly check. So I though the documentation can be fixed once the code base is more thorough.